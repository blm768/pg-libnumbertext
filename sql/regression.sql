CREATE EXTENSION IF NOT EXISTS pg_libnumbertext;

SELECT numbertext('1', 'en');