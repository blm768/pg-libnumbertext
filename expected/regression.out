CREATE EXTENSION IF NOT EXISTS pg_libnumbertext;
SELECT numbertext('1', 'en');
 numbertext 
------------
 one
(1 row)

SELECT numbertext('3', 'ja');
 numbertext 
------------
 ä¸‰
(1 row)

SELECT numbertext('not a number', 'en');
 numbertext 
------------
 
(1 row)

SELECT numbertext('1', 'not a language');
ERROR:  Failed to load Soros code for language "not a language"
